agenda-api/src/main/java/org/exoplatform/agenda/service/AgendaEventAttendeeService.java

```diff
@@ -41,8 +41,9 @@
    * 
    * @param eventId technical identifier of the event
    * @param isNew flag to indicate if event is new or old
+   * @param isDeleted flag to indicate if event is deleted or not
    */
-  public void sendInvitations(long eventId, boolean isNew);
+  public void sendInvitations(long eventId, boolean isNew, boolean isDeleted);
```

Comment:
Instead of adding a new parameter, you can add an enum that defines the modification type here, for example :
```
enum EventModificationType {
  ADDED, UPDATED, DELETED
}
```
this way, you can pass one single parameter, you make the API more evolutive ( if we will handle more modification types, such as a modification by specifying field name )...